corehq/apps/geospatial/templates/map_visualization.html

```diff
@@ -95,7 +95,20 @@
     {% trans "Running disbursement algorithm..." %}
   </h4>
 </div>
-<div id="disbursement-error" class="alert alert-danger" data-bind="visible: disbursementErrorMessage(), html: disbursementErrorMessage">
+<div id="disbursement-error">
+  <div class="alert alert-danger" data-bind="visible: disbursementErrorMessage(), html: disbursementErrorMessage">
+  </div>
+  <div class="alert alert-danger" data-bind="visible: showUnassignedCasesError()">
+    {% blocktrans %}
+    We couldn't match every case to a user with the current disbursement settings.
+    Please follow any of the below steps to rectify this.
+    <ul>
+      <li>Ensure that your settings are correct</li>
+      <li>Allocate more users to the area</li>
```

Comment:
Can we easily tell if the cause of error was from left over cases that couldn't be assigned due to a lack of mobile workers? If so, it could be nice to only show point 2 and 3 if this condition is true. This is just a thought, so feel free to ignore and keep the banner as is.

Reply:
Well, is "left over cases" due to too much cases or too little CHWs? We don't know, and that's up for the project to decide.